1.0/docs/start/psk/setting-up.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Setting up
+subtitle: Polymer Starter Kit
+---
+
+## Setting up the Polymer Starter Kit
```

Comment:
Applies to all page titles & headings: we try not to use gerunds, so go with "Set up ...," "Create a new page," etc.